README.md

```diff
@@ -315,7 +315,7 @@ module.exports = {
         test: /\.lazy\.css$/i,
         use: [
           { loader: 'style-loader', options: { injectType: 'lazyStyleTag' } },
-          'css-loader',
+          { loader: 'css-loader', options: { modules: true } },
```

Comment:
Laze can be used without css modules

Reply:
Of course it will work, but there will be no `locals` if `modules` will be falsy.
This example has `divElement.className = styles.locals['my-class'];` which will not work without `modules: true`.